src/main/java/net/sf/rails/tools/ListAndFixSavedFiles.java

```diff
@@ -512,6 +506,37 @@ PossibleAction processInput() {
         }
     }
 
+    private class BuyCertificateDialog extends EditDialog {
```

Comment:
I think this class should be moved to its own `.java` file. The file is becoming a bit too long

Reply:
Generally, I don't bother much with the length of java files. 
In this case, we are talking about a simple tool that does not even has its own package, but coexists with more such tools. Splitting off separate .java files would be confusing, unless we create separate packages for such stuff.
For me, this has priority zero.